### ISSUE TITLE
Licensing?

### ISSUE THREAD

**Issue by chroby** [1ec5/hunspell-vi#12]
The dictionaries do have a README outlining their licensing, but it would probably be useful to have a top-level LICENSE.md

**Comment by 1ec5** [comment]
Agreed. It may make sense to license under GPLv2 like the dictionaries, just to keep things simple. However, I generally prefer to use a permissive license when it comes to language tools (like the ones in tools/) that are often reused in unforeseen ways. I'm open to suggestions, particularly from @clytie, who contributed the OpenOffice.org part of this project.

**Comment by throwaway-d** [comment]
Xin bạn hãy thêm sớm để khỏi phải đối mặt với những vấn đề pháp lý sau này! Please add soon so we don't have to deal with legal issues in the future!
